packages/docs/src/components/DocsCode.vue

```diff
@@ -16,10 +26,36 @@ export default defineComponent({
   components: { PrismWrapper },
   props: {
     language: { type: String as PropType<string>, default: 'javascript' },
-    code: { type: String as PropType<string>, default: '' },
+    code: { type: [Object, String] as PropType<Record<string, string> | string>, default: '' },
   },
   setup (props) {
-    const formattedCode = computed(() => applyTranslations(props.code.replace(/^\n|\n$/g, '')))
+    const isString = computed(() => typeof props.code === 'string')
+
+    const tabs = computed(() => {
+      const result: Array<string> = []
```

Comment:
You can replace `Array<string>` with `string[]`. Does the same, looks better :).

Reply:
Absolutely.